### PR TITLE
Troubleshooting cm

### PR DESCRIPTION
**Pull request by cristinamullin**

### CODE REVIEW

File: R/DataDiscoveryRetrieval.R [hunk 1]
```diff
@@ -52,14 +56,32 @@
...
+#' tada3 <- TADAdataRetrieval(statecode = "UT", 
+#'                            characteristicName = c("Ammonia", "Nitrate", "Nitrogen"), 
+#'                            startDate = "10-01-2020")
+#' 
+#' tada4 <- TADAdataRetrieval(statecode = "SC", countycode  = "Abbeville")
+#' 
+#' # countycode queries require a statecode
+#' tada5 <- TADAdataRetrieval(countycode = "US:02:020")
+#' 
+#' tada6 <- TADABigdataRetrieval(startDate = "2019-01-01",
```
Comment (ehinman) [inline R/DataDiscoveryRetrieval.R:71]: Should change date format so it is consistent across examples. Let me know if you'd like me to do this on my next branch commit.

**Review by ehinman — APPROVED** [review]
All looks good to me and runs well on my computer. I think I left a single comment about making date format consistent--looks like I switched it up in TADABigdataRetrieval. Let me know how you'd like to proceed.

**Review by ehinman — COMMENTED** [review]
Do you still want a review? It all looks fine to me!

**Comment by cristinamullin** [comment]
No, all set. This branch has been merged and closed. Thanks!

**Pull request closed without merging**